src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ToggleSwitchTests/UnoSamples_Tests.ToggleSwitch.cs

```diff
@@ -37,5 +37,28 @@ public void ToggleSwitch_TemplateReuseTest()
 			Assert.AreEqual("False", toggleSwitchGroup.GetDependencyPropertyValue("IsOn")?.ToString());
 			Assert.AreEqual("True", separatedToggleSwitch.GetDependencyPropertyValue("IsOn")?.ToString());
 		}
+
+		[Test]
+		[AutoRetry]
+		public void ToggleSwitch_HeaderTest()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.ToggleSwitchControl.ToggleSwitch_Header");
+
+			var toggleSwitchWithHeader = _app.Marked("toggleSwitchWithHeader");
+			_app.WaitForElement(toggleSwitchWithHeader);
+
+			var toggleSwitchHeader = toggleSwitchWithHeader.Marked("toggleSwitchWithHeader");
+			_app.WaitForElement(toggleSwitchHeader);
```

Comment:
This is not needed, as `toggleSwitchWithHeader` cannot be named in the XAML (it's a property, not an actual visual tree element).
```suggestion
```